ckan/templates-midnight-blue/package/resource_views.html

```diff
@@ -23,6 +23,7 @@
 {% endblock %}
 
 {% block primary_content_inner %}
+  <h1>{{ _('Edit views') }}</h1>
```

Comment:
we're not consistent about capitalization of titles in ckan but I believe title case is recommended for these, in English at least.
```suggestion
  <h1>{{ _('Edit Views') }}</h1>
```